lib/entities/request/ClassicRequest.ts

```diff
@@ -24,6 +24,7 @@ export interface ClassicConfig {
   debugRoutingConfig?: string;
   unicornSecret?: string;
   quoteSpeed?: string;
+  enableFeeOnTransferFeeFetching?: boolean;
```

Comment:
btw just curious, UniswapX just handles the fees but we wont show it as extra fees right?

Reply:
Aw yes, my understanding is that UniswapX is based off of quoteAmountGasAdjusted from classic (https://github.com/Uniswap/unified-routing-api/blob/main/lib/handlers/quote/handler.ts#L381). Since in sor our final v2 quoteAmountGasAdjusted will be after subtracting FOT taxes, due to the changes in https://github.com/Uniswap/v2-sdk/pull/144, UniswapX will have to be better than classic.quoteAmountGasAdjusted. Since fillers also have to quote themselves, dutch.amountOut is also after subtracting FOT taxes.